.travis.yml

```diff
@@ -14,7 +14,7 @@ env:
   - TOX_ENV=py34-lib
   - TOX_ENV=py34-examples
 script:
-  - tox -e $TOX_ENV
+  - tox -e $TOX_ENV -- pip install eventlet==0.20.0
```

Comment:
0.20.1 is broken on several OS's unfortunately, including the build environment here -- https://github.com/eventlet/eventlet/issues/401